src/llama_recipes/utils/train_utils.py

```diff
@@ -66,6 +67,11 @@ def train(model, train_dataloader,eval_dataloader, tokenizer, optimizer, lr_sche
     checkpoint_times = []
     results = {}
     best_val_loss = float("inf")
+
+    track_metrics = True
+    if train_config.enable_fsdp and rank != 0:
+        track_metrics = False
```

Comment:
just `track_metrics = ....` in one line is a bit cleaner. something like `track_metrics = not fsdp or rank == 0` I think would be equivalent.

Reply:
done